IronSoftware.Drawing/IronSoftware.Drawing.Common.Tests/UnitTests/AnyBitmapFunctionality.cs

```diff
@@ -407,6 +409,19 @@ public void CastSixLabors_from_AnyBitmap()
             AssertImageAreEqual("expected.bmp", "result.bmp", true);
         }
 
+        [FactWithAutomaticDisplayName]
+        public void CastBitmap_to_AnyBitmap_using_FromBitmap()
+        {
+            string imagePath = GetRelativeFilePath("van-gogh-starry-night-vincent-van-gogh.jpg");
+            var imgSharp = Image.Load<Rgba32>(imagePath);
```

Comment:
I think better to use `Bitmap` to load original to check instead using `ImageSharp`. Because we try to test casting and loading using `Bitmap`.